test/unit/cameras/PerspectiveCamera.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @author simonThiele / https://github.com/simonThiele
+ */
+
+module( "PerspectiveCamera" );
+
+test( "updateProjectionMatrix", function() {
+	var near = 1,
+			far = 3,
+			bottom = -1,
+			top = 1,
```

Comment:
> In my understanding, none of these values (l, r, t, b, n, f) depending on fov 

Then you are confused. Step through the code with the debugger.

Just do something really simple. You were on the right track.

    var cam = new THREE.PerspectiveCamera( fov, aspect, near, far );
    var m = cam.projectionMatrix; // set in the constructor
    // now test the values against a known answer, which you compute from the 4 inputs to the constructor.
    m.equals( correctMatrix );